macros/JsSidebar.ejs

```diff
@@ -394,102 +394,144 @@ var text = mdn.localStringMap({
  <ol>
   <li><a href="/<%=locale%>/docs/Web/JavaScript"><strong><em>JavaScript</em></strong></a></li>
   <li><a href="/<%=locale%>/docs/Web/JavaScript/Tutorials"><strong><%=text['Tutorials']%></strong></a></li>
-  <li data-default-state="<%=state('Complete_beginners')%>"><a href="#"><%=text['Complete_beginners']%></a>
-   <ol>
-    <li><a href="/<%=locale%>/docs/Learn/Getting_started_with_the_web/JavaScript_basics"><%=text['Basics']%></a></li>
-    <li><a href="/<%=locale%>/docs/Learn/JavaScript/First_steps"><%=text['First_steps']%></a></li>
-    <li><a href="/<%=locale%>/docs/Learn/JavaScript/Building_blocks"><%=text['Building_blocks']%></a></li>
-    <li><a href="/<%=locale%>/docs/Learn/JavaScript/Objects"><%=text['Introducing_objects']%></a></li>
-   </ol>
+  <li class="toggle">
```

Comment:
The ``{{JSSidebar}}`` macro is huge. It also appears to be rendered correctly on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide 👍.

<img width="534" alt="pr 789 jssidebar" src="https://user-images.githubusercontent.com/286017/45384772-8f7f9f00-b5d4-11e8-81a6-c291a32ba1ea.png">